encode.go

```diff
@@ -287,6 +287,11 @@ func (en *encoder) slice(key uint64, slval reflect.Value) {
 			packed.uvarint(v)
 		}
 
+	case []int:
+		for i := 0; i < sllen; i++ {
+			packed.svarint(int64(slt[i]))
+		}
+
 	case []int32:
 		for i := 0; i < sllen; i++ {
 			packed.svarint(int64(slt[i]))
```

Comment:
Mhh i was not aware we use `[]int64` when we want to send `[]int32`, maybe we should change that too to stay consistent so developers can still use this library even if it only supports 32 bits integers ?